CONTRIBUTING.md

```diff
@@ -53,6 +53,47 @@ npm run serve         # Uses the .env.prod config file
 npm run serve:staging # Uses the .env.staging config file
 ```
 
+### Develop with local firebase emulators
+
+If you have already configured the firebase CLI you can skip the first two steps.
+
+1. Install Firebase CLI
+
+```
+npm i -g firebase-tools
+
+```
+
+2. Sign in to use the CLI
+
+```
+firebase login
+```
+
+3. Initialize emulators
+
+```
+firebase init emulators
```

Comment:
Eg trur dette er berre nødvendig første gong for å sette opp firebase.json riktig med emulators-feltet. Eg trur neste steg (`firebase emulators:start`) vil installere manglande emulatorar basert på firebase.json. Men det går kanskje an å teste?

Reply:
Ja, det gir mening! Skal sjekke det ut:)

Reply:
Så ut som det du foreslo stemte. Reinstallerte firebase-tools og trengte ikke å kjøre noe init for at emulatorene skulle dure i gang. Oppdatert readme med det.